zeppelin-web/Gruntfile.js

```diff
@@ -55,15 +55,15 @@ module.exports = function(grunt) {
           expand: true,
           cwd: './src/',
           src: ['**/*.js'],
-          dest: '.tmp',
```

Comment:
@felizbear Good point.
JSLint of My editor pointed this out as an error and modified it. I had never thought before that this would be helpful.
I have read a [ESLint post](http://eslint.org/docs/rules/comma-dangle) in this regard, and it makes a lot of sense.

However, there are more places in this file without a dangled comma. I think it should be applied to the whole file equally for consistency.

What do you think? Do I have to change everything else?